app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx

```diff
@@ -111,6 +126,110 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
     fetchPolicy: "no-cache",
   })
 
+  const {
+    loading,
+    data: contactsData,
+    error,
+  } = useContactsQuery({
+    skip: !isAuthed,
+    fetchPolicy: "cache-and-network",
+  })
+
+  if (error) {
+    toastShow({ message: error.message, LL })
+  }
```

Comment:
have you tested this use case? I think it will likely loop indefinitely, ie: retrigger the toastShow message on every render. I would guess this might not be the intent.

Reply:
Was taken directly from [screens/people-screen/contacts/all-contacts.tsx](https://github.com/GaloyMoney/galoy-mobile/blob/84eaa80cde72b5d8fc0ad65402066ff1ed6b72d0/app/screens/people-screen/contacts/all-contacts.tsx#L53-L55) but together with the useContactsQuery. 
Removed it, let me know what you think

Reply:
ok I guess this should have been manually tested then, and I may have this logic wrong. closing it.